app/views/producers/index.html.haml

```diff
@@ -1,31 +1,32 @@
-- content_for(:title) do
-  = t :producers_title
+- cache 'cached_producers_page_content', expires_in: 30.minutes do
```

Comment:
I put 30 minutes but we can adjust it for sure.

Reply:
I was not that inspired for the cache key ever... Well, let's validate the approach first.

Reply:
Take a look at https://app.datadoghq.com/logs/analytics?agg_q=&cols=core_host%2Ccore_service%2Clog_response_time&from_ts=1586152800000&index=main&live=false&messageDisplay=inline&query=%40http.url_details.path%3A%28%22%2Fshops%22%20OR%20%22%2Fshops%2F%22%29%20host%3Awww.openfoodnetwork.org.uk&step=300000&stream_sort=desc&to_ts=1586188020000&top_n=5&viz=timeseries. If we go with 5min we would save around 10reqs every 5min. At 13:30h we would have performed just 1 out of 38 requests.